locales/ru_RU.json

```diff
@@ -1,104 +1,104 @@
 {
   "sandbox-warning": "This is a sandbox environment for testing API integration. Never send real funds here!",
-  "buy": "buy",
-  "sell": "sell",
-  "continue": "Continue",
-  "ok": "OK",
-  "back": "Back",
-  "help": "Help",
-  "and": "and",
-  "from": "From",
-  "to": "To",
-  "or": "or",
-  "price": "Price",
-  "fee": "Fee",
-  "try-again": "Try again",
-  "refresh": "Refresh",
-  "finish": "Finish",
-  "price-each": "{{price}} each",
-  "exchange-rate": "<0 /> / <1>{{sourceCurrency}}</1>",
-  "remaining": "Remaining",
+  "buy": "Покупка",
+  "sell": "Продаж",
+  "continue": "Продожать",
+  "ok": "Окей",
+  "back": "назад",
+  "help": "помощь",
+  "and": "и",
+  "from": "с",
+  "to": "в/на",
+  "or": "или",
+  "price": "Прайс",
+  "fee": "Сбор",
+  "try-again": "Попробуйте еще раз",
+  "refresh": "Обновление",
+  "finish": "Окончание",
+  "price-each": "{{цена}} каждая",
+  "exchange-rate": "<0 /> / <1>{{источник валюта}}</1>",
```

Comment:
@buyfromchina Could you please review the README? You are translating the text in `{{ }}` which you should not change.
https://github.com/nash-io/locale/blob/master/README.md